MonoGame.Framework/Graphics/Effect/SpriteEffect.cs

```diff
@@ -1,24 +1,12 @@
-#region File Description
-//-----------------------------------------------------------------------------
-// SpriteEffect.cs
-//
 // Microsoft XNA Community Game Platform
 // Copyright (C) Microsoft Corporation. All rights reserved.
-//-----------------------------------------------------------------------------
```

Comment:
If we add our license header, it should be in addition to the unmodified original.

```
// MonoGame - Copyright (C) The MonoGame Team
// This file is subject to the terms and conditions defined in
// file 'LICENSE.txt', which is part of this source code package.
//
// based on
//-----------------------------------------------------------------------------
// SpriteEffect.cs
//
// Microsoft XNA Community Game Platform
// Copyright (C) Microsoft Corporation. All rights reserved.
//-----------------------------------------------------------------------------
```